scylladb/sphinx-scylladb-theme
# Add zendesk tag support

This PR lets projects specify custom Zendesk tags in their `conf.py` file as follows:

```
html_theme_options = {
    'zendesk_tag': 'gq6ltsh3nfex3cnwfy4aj9',
}
```

Different from the previous approach, the tag is added to the main `index.html` file that handles redirections.

# How to test

1. Run `make multiversion`.
1. Open the `_build/dirhtml/index.html` file.
1. Check for the presence of the Zendesk tag associated with this project:

   
    <img width="773" alt="image" src="https://github.com/scylladb/sphinx-scylladb-theme/assets/9107969/171d064a-fe7b-4165-8ad5-e8f1489638b3">
